client/content/banqi.css

```diff
@@ -105,14 +105,21 @@ html:before {
 .dead-pieces {
   height: auto;
 }
-.dead-pieces .banqi-square{
+.dead-pieces .banqi-square {
   height:45px;
   width:45px;
   background-repeat:no-repeat;
   background-size:cover;
   margin:2px;
   display:inline-block;
 }
+.dead-pieces .banqi-square.live {
+    opacity: 20%;
+}
+.dead-pieces .banqi-square.unborn {
+  opacity: 20%;
+  background-image: url("/images/unflipped-piece.svg");
+}
```

Comment:
```suggestion

```

Reply:
I think this change will get closer to the image, and it gets rid of the distracting opacity issues.  It even crops the effect to the circle of the piece.  Thoughts?

Reply:
I think that looks good. Here's what I'm seeing:
![Screen Shot 2020-05-30 at 6 22 58 PM](https://user-images.githubusercontent.com/335065/83341625-a3849c00-a2a2-11ea-8bdd-38467fd20175.png)